_posts/2020-10-23-kubernetes-webhooks-made-easy-with-openfaas.md

```diff
@@ -0,0 +1,265 @@
+---
+title: "Kubernetes Webhooks made easy with OpenFaaS"
+description: "Extending the Kubernetes with Admission webhook using OpenFaaS Serverless Functions"
+date: 2020-10-23
```

Comment:
Put the date to 27th October please and rename the file with `git mv` so the newer date.

Reply:
done.